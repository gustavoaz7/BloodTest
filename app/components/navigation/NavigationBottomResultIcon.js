import React from 'react';
// eslint-disable-next-line import/no-extraneous-dependencies
import { Feather } from '@expo/vector-icons';
import { tabBarIconTypes } from '../../types';

export default function NavigationBottomResultIcon({ tintColor }) {
  return <Feather name="bar-chart-2" size={26} color={tintColor} />;
}

NavigationBottomResultIcon.propTypes = tabBarIconTypes;
