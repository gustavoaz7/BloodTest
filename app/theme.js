export const theme = {
  primary: '#f5785a',
  secondary: '#321b3c',
  fontFamily: 'PostGrotesk-Light',
};
