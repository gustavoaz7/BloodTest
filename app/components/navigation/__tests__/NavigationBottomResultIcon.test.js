import React from 'react';
import { shallow } from 'enzyme';
import NavigationBottomResultIcon from '../NavigationBottomResultIcon';
import { theme } from '../../../theme';

describe('NavigationBottomResultIcon', () => {
  it('renders without crashing', () => {
    shallow(<NavigationBottomResultIcon />);
  });

  it('accepts `tintColor`', () => {
    const wrapper = shallow(<NavigationBottomResultIcon tintColor={theme.primary} />);
    expect(wrapper.prop('color')).toBe(theme.primary);
  });

  it('matches snapshot', () => {
    const wrapper = shallow(<NavigationBottomResultIcon tintColor={theme.primary} />);
    expect(wrapper).toMatchSnapshot();
  });
});
