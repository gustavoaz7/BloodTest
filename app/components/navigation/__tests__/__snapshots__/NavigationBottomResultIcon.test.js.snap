// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavigationBottomResultIcon matches snapshot 1`] = `
<Icon
  allowFontScaling={false}
  color="#f5785a"
  name="bar-chart-2"
  size={26}
/>
`;
